cpp/ql/test/library-tests/identity_string/identity_string.cpp

```diff
@@ -177,7 +177,7 @@ namespace Outer {
         struct LocalClassInLambda {
           void MemberFuncOfLocalClassInLambda() { }
         };
-        check_func(&LocalClassInLambda::MemberFuncOfLocalClassInLambda, "void (int (int Outer::Inner::globalFunctionTemplate<long, double>(long, double))::(lambda [] type at line ?, col. ?)::operator()(int) const)::LocalClassInLambda::MemberFuncOfLocalClassInLambda()");
+        check_func(&LocalClassInLambda::MemberFuncOfLocalClassInLambda, "void (int (int Outer::Inner::globalFunctionTemplate<long, double>(long, double))::(lambda [])::operator()(int) const)::LocalClassInLambda::MemberFuncOfLocalClassInLambda()");
```

Comment:
Now you can also delete https://github.com/Semmle/ql/blob/566eafc7060a3481b462d6b2f1e1ed887c354252/cpp/ql/test/library-tests/identity_string/identity_string.ql#L78-L80

Reply:
Have changed the patch to keep the line and column number in the names, so this is still necessary.